CMakeLists.txt

```diff
@@ -481,7 +481,7 @@ export(PACKAGE SMTK)
 #
 # Required to happen before we include the docs directory
 set(MOAB_ROOT_DIR "${MOAB_ROOT_DIR}")
-set(SMTK_MODULE_DIR "${SMTK_SOURCE_DIR}/CMake")
+set(SMTK_MODULE_DIR "${CMAKE_BINARY_DIR}/lib/cmake/SMTK")
```

Comment:
It's the directory where VTK is putting the generated module info files. And the patch for where these files can be found is dictated by CMake.

Putting the version into it would mean changing the `vtk_smtk_setup_module_environment` calls to have it.